### PR TITLE
reference/performance: add description for stats healthy

### PR DESCRIPTION
**Pull request by lamxTyler**
### What is changed, added or deleted? 

Fix https://github.com/pingcap/docs-cn/issues/2067

### What is the related PR or file link(s)? 

N/A

### Which version does your change affect? 

dev/v3.1/v3.0/v2.1

### CODE REVIEW

File: dev/reference/performance/statistics.md [hunk 1]
```diff
@@ -180,6 +180,29 @@ SHOW STATS_META [ShowLikeOrWhere];
...
+
+语法如下：
+
+{{< copyable "sql" >}}
+
+```sql
+SHOW STATS_HEALTHY [ShowLikeOrWhere];
+```
+
+该语句会输出所有表的健康度信息，可以通过 ShowLikeOrWhere 来筛选需要的信息。
```
Comment (juliezhang1112) [inline dev/reference/performance/statistics.md]: Let's put the sentence before the code block.

File: dev/reference/performance/statistics.md [hunk 2]
```diff
@@ -180,6 +180,29 @@ SHOW STATS_META [ShowLikeOrWhere];
...
+
+目前 `SHOW STATS_HEALTHY` 会输出 4 列，具体如下：
```
Comment (juliezhang1112) [inline dev/reference/performance/statistics.md]:
```suggestion
目前，`SHOW STATS_HEALTHY` 会输出以下 4 项内容：
```

File: dev/reference/performance/statistics.md [hunk 3]
```diff
@@ -180,6 +180,29 @@ SHOW STATS_META [ShowLikeOrWhere];
...
+
+| 语法元素 | 说明            |
+| -------- | ------------- |
```
Comment (juliezhang1112) [inline dev/reference/performance/statistics.md]:
```suggestion
| :-------- | :------------- |
```

File: dev/reference/performance/statistics.md [hunk 4]
```diff
@@ -180,6 +180,27 @@ SHOW STATS_META [ShowLikeOrWhere];
 >
 > 在 TiDB 根据 DML 语句自动更新总行数以及修改的行数时，`update_time` 也会被更新，因此并不能认为 `update_time` 是最近一次发生 Analyze 的时间。
 
+### 表的健康度信息
+
+通过 `SHOW STATS_HEALTHY` 可以查看表的统计信息健康度，并粗略估计表上统计信息的准确度。当 `modify_count` >= `row_count` 时，健康度为 0，当 `modify_count` < `row_count` 时，健康度为 (1 - `modify_count`/`row_count`) * 100。
```
Comment (lilin90) [inline dev/reference/performance/statistics.md]:
```suggestion
通过 `SHOW STATS_HEALTHY` 可以查看表的统计信息健康度，并粗略估计表上统计信息的准确度。当 `modify_count` >= `row_count` 时，健康度为 0；当 `modify_count` < `row_count` 时，健康度为 (1 - `modify_count`/`row_count`) * 100。
```

File: v2.1/reference/performance/statistics.md [hunk 5]
```diff
@@ -98,6 +98,26 @@ SHOW STATS_META [ShowLikeOrWhere]
 >
 > 在 TiDB 根据 DML 语句自动更新总行数以及修改的行数时，`update_time` 也会被更新，因此并不能认为 `update_time` 是最近一次发生 Analyze 的时间。
 
+### 表的健康度信息
+
+通过 `SHOW STATS_HEALTHY` 可以查看表的统计信息健康度，并粗略估计表上统计信息的准确度。当 `modify_count` >= `row_count` 时，健康度为 0，当 `modify_count` < `row_count` 时，健康度为 (1 - `modify_count`/`row_count`) * 100。
```
Comment (lilin90) [inline v2.1/reference/performance/statistics.md]:
```suggestion
通过 `SHOW STATS_HEALTHY` 可以查看表的统计信息健康度，并粗略估计表上统计信息的准确度。当 `modify_count` >= `row_count` 时，健康度为 0；当 `modify_count` < `row_count` 时，健康度为 (1 - `modify_count`/`row_count`) * 100。
```

File: v3.0/reference/performance/statistics.md [hunk 6]
```diff
@@ -181,6 +181,27 @@ SHOW STATS_META [ShowLikeOrWhere];
 >
 > 在 TiDB 根据 DML 语句自动更新总行数以及修改的行数时，`update_time` 也会被更新，因此并不能认为 `update_time` 是最近一次发生 Analyze 的时间。
 
+### 表的健康度信息
+
+通过 `SHOW STATS_HEALTHY` 可以查看表的统计信息健康度，并粗略估计表上统计信息的准确度。当 `modify_count` >= `row_count` 时，健康度为 0，当 `modify_count` < `row_count` 时，健康度为 (1 - `modify_count`/`row_count`) * 100。
```
Comment (lilin90) [inline v3.0/reference/performance/statistics.md]:
```suggestion
通过 `SHOW STATS_HEALTHY` 可以查看表的统计信息健康度，并粗略估计表上统计信息的准确度。当 `modify_count` >= `row_count` 时，健康度为 0；当 `modify_count` < `row_count` 时，健康度为 (1 - `modify_count`/`row_count`) * 100。
```

File: v3.1/reference/performance/statistics.md [hunk 7]
```diff
@@ -180,6 +180,27 @@ SHOW STATS_META [ShowLikeOrWhere];
 >
 > 在 TiDB 根据 DML 语句自动更新总行数以及修改的行数时，`update_time` 也会被更新，因此并不能认为 `update_time` 是最近一次发生 Analyze 的时间。
 
+### 表的健康度信息
+
+通过 `SHOW STATS_HEALTHY` 可以查看表的统计信息健康度，并粗略估计表上统计信息的准确度。当 `modify_count` >= `row_count` 时，健康度为 0，当 `modify_count` < `row_count` 时，健康度为 (1 - `modify_count`/`row_count`) * 100。
```
Comment (lilin90) [inline v3.1/reference/performance/statistics.md]:
```suggestion
通过 `SHOW STATS_HEALTHY` 可以查看表的统计信息健康度，并粗略估计表上统计信息的准确度。当 `modify_count` >= `row_count` 时，健康度为 0；当 `modify_count` < `row_count` 时，健康度为 (1 - `modify_count`/`row_count`) * 100。
```

**Pull request merged**